Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -91,28 +88,38 @@ void setInitialTime(void)
   sDate.Date = 1;
   sDate.Year = 23;
   HAL_RTC_SetDate(&hrtc, &sDate, RTC_FORMAT_BIN);
```

Comment:
Try to understand 3rd argument of **HAL_RTC_SetDate**, what is **RTC_FORMAT_BIN**? and what are other possibilities for   HAL_RTC_SetDate 3rd argument.